src/misc/camera.ts

```diff
@@ -2,50 +2,25 @@ import { StreamApiNotSupportedError, InsecureContextError, StreamLoadTimeoutErro
 import { eventOn, timeout } from './callforth'
 import shimGetUserMedia from './shimGetUserMedia'
 
-type CameraActive = {
-  isActive: true
-  torchOn: boolean
-  videoEl: HTMLVideoElement
-  stream: MediaStream
-}
-
-type CameraInactive = { isActive: false }
-
-type Camera = CameraActive | CameraInactive
-
-let cameraState: Camera = { isActive: false }
-
-export async function stop() {
-  if (cameraState.isActive) {
-    if (cameraState.torchOn) {
-      // @ts-ignore
-      await track.applyConstraints({ advanced: [{ torch: false }] })
```

Comment:
Ah only now realize that I don't actually have access to `track` here and the `@ts-ignore` probably prevented me from seeing this. Was that the issue with torch not turning off?

Reply:
This is the error thrown in the console on my Android phone, and is the reason I augment the types to avoid using `@ts-ignore`. I refactored since then so I cannot be sure if this is the only problem.